pydarn/plotting/rtp.py

```diff
@@ -6,13 +6,16 @@
 """
 Range-time Intensity plots
 """
-import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib import dates, colors, cm, ticker
+import numpy as np
 from typing import List
-from datetime import datetime
+from datetime import datetime, timedelta
 
```

Comment:
The above packages should be listed alphabetically, I brelieve that just requires moving datetime to the top.

Reply:
That is a preference of style, PEP8 does not suggest anything about it. But we can do it and I will have to remember to place the preference in the style guide.

Reply:
Done and did it in dmap.py and superdarn.py